### PR TITLE
Feat: APP-2147 Remove search input from Transfers page

### PR DESCRIPTION
**Pull request by cgero-eth**
## Description

- Remove the `SearchInput` component from the Transfer page
- Remove now unused `searchTransfers` localisation key
- Update `eqeqeq` eslint rules to allow null check
  - Eslint reference: [allow-null](https://eslint.org/docs/latest/rules/eqeqeq#allow-null)
  - I find the expression `x == null` valuable to check if a variable is null or undefined

Task: [APP-2147](https://aragonassociation.atlassian.net/browse/APP-2147)

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-2147]: https://aragonassociation.atlassian.net/browse/APP-2147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/components/verificationCard/index.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ const VerificationCard: React.FC<TransferListProps> = ({tokenAddress}) => {
                   <Dd>{tokenTotalHolders}</Dd>
                 ) : (
                   <dd className="flex items-center" style={{width: '70%'}}>
-                    <IconSpinner className="w-1.5 desktop:w-2 h-1.5 desktop:h-2 animate-spin text-primary-500" />
+                    <IconSpinner className="w-1.5 desktop:w-2 h-1.5 desktop:h-2 text-primary-500 animate-spin" />
```
Comment (Fabricevladimir) [inline packages/web-app/src/components/verificationCard/index.tsx:162]: The bane of my existence; I have personally fixed this on 3 different PRs -_-

**Review by Fabricevladimir — APPROVED** [review]
Thanks, Ruggero! Looking good

File: packages/web-app/src/components/verificationCard/index.tsx (hunk 1 above)
Comment (cgero-eth) [reply packages/web-app/src/components/verificationCard/index.tsx:162]: Ahaha I see, might be a ESLint config mismatch between local & remote, will try to figure it out 😄

Comment (Fabricevladimir) [reply packages/web-app/src/components/verificationCard/index.tsx:162]: Might be, no worries though!

**Pull request merged**